data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaSourceCustomConsumer.java

```diff
@@ -176,19 +180,42 @@ public <T> void consumeRecords() throws Exception {
             topicMetrics.getNumberOfPollAuthErrors().increment();
             Thread.sleep(10000);
         } catch (RecordDeserializationException e) {
-            LOG.warn("Deserialization error - topic {} partition {} offset {}",
-                    e.topicPartition().topic(), e.topicPartition().partition(), e.offset());
+
+            LOG.warn("Deserialization error - topic {} partition {} offset {}. Error message: {}",
+                    e.topicPartition().topic(), e.topicPartition().partition(), e.offset(), e.getMessage());
             if (e.getCause() instanceof AWSSchemaRegistryException) {
-                LOG.warn("AWSSchemaRegistryException: {}. Retrying after 30 seconds", e.getMessage());
+                LOG.warn("Retrying after 30 seconds");
```

Comment:
Why would we not want to show the exception message here?

Reply:
It is logged on line 185 now.